### ISSUE TITLE
Test Accuracy

### ISSUE THREAD

**Issue by hirunifernando** [RayanWang/Speech_emotion_recognition_BLSTM#6]
Hi RayanWang,
I have started the model training with the use of find_best_model.py.Below validation accuracy was achieved.
While the training process is ongoing 200 epochs, highest validation accuracy was 0.5652(early stopping at 35)

While the training process is ongoing 100 epochs, highest validation accuracy was 0.3354(early stopping at 19)
I want to increase the testing accuracy more (at least 70).what are additional things and modifications should I follow.
Thank u.

**Comment by RayanWang** [comment]
First of all, these two results may sound strange. Maybe depend on your optimizer or other hyper parameter you tuned. I recommend using cross-validation first to confirm if the data set is sufficient and correct.

**Comment by hirunifernando** [comment]
After finishing the training process, max accuracy was achieved as 58. I want a high accuracy model to predict emotions. So  If u can plz send me the trained model which have accuracy around 68.
Thank u for ur support
